src/View/Sidebar/AbstractMountableRow.vala

```diff
@@ -123,33 +124,32 @@ public abstract class Sidebar.AbstractMountableRow : Sidebar.BookmarkRow, Sideba
             reveal_child = false
         };
 
-        unmount_eject_revealer.add (unmount_eject_button);
+        unmount_eject_revealer.set_child (unmount_eject_button);
 
         unmount_eject_working_stack = new Gtk.Stack () {
             margin_start = 6,
             transition_type = Gtk.StackTransitionType.CROSSFADE
         };
 
-        unmount_eject_working_stack.add (unmount_eject_revealer);
-        unmount_eject_working_stack.add (working_spinner);
+        unmount_eject_working_stack.add_child (unmount_eject_revealer);
+        unmount_eject_working_stack.add_child (working_spinner);
 
         content_grid.attach (unmount_eject_working_stack, 1, 0);
 
         storage_levelbar = new Gtk.LevelBar () {
             value = 0.5,
-            hexpand = true,
-            no_show_all = true
+            hexpand = true
         };
         storage_levelbar.add_offset_value (Gtk.LEVEL_BAR_OFFSET_LOW, 0.9);
         storage_levelbar.add_offset_value (Gtk.LEVEL_BAR_OFFSET_HIGH, 0.95);
         storage_levelbar.add_offset_value (Gtk.LEVEL_BAR_OFFSET_FULL, 1);
 
         unowned var storage_style_context = storage_levelbar.get_style_context ();
-        storage_style_context.add_class (Gtk.STYLE_CLASS_FLAT);
+        storage_style_context.add_class ("flat");
         storage_style_context.add_class ("inverted");
         storage_style_context.add_provider (devicerow_provider, Gtk.STYLE_PROVIDER_PRIORITY_APPLICATION);
 
-        icon_label_grid.attach (storage_levelbar, 1, 1);
+        icon_label_grid.append (storage_levelbar);
```

Comment:
This is intentionally an attach because the icon has a height of 2 units

Reply:
Reverted